.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,47 @@
+name: Ubuntu CI
+
+on:
+  push:
+    branches-ignore:
+      # Exclude the push event for exported diffs, because the CI for export
+      # should have been covered by GitHub Actions triggered by pull requests.
+      - 'export-D+'
+  pull_request:
+
+concurrency:
+  # If the workflow is triggered by a pull request, then cancel previous runs
+  # for the same pull request, which share the same `github.ref`, otherwise the
+  # run ID is used to identify the concurrency group, which is a no-op because
+  # the run ID is always unique for each trigged event.
+  group: ${{ github.event_name == 'pull_request' && github.ref || github.run_id }}
```

Comment:
I think this is leading to the Ubuntu CI job being cancelled - from https://github.com/facebook/hhvm/actions/runs/3732328240 :

> Canceling since a higher priority waiting request for 'refs/pull/9306/merge' exists

Probably want something along the lines of:

```
  group: ubuntu-ci-${{ github.event_name == 'pull_request' && github.ref || github.run_id }}
```

Reply:
The nix failures seem bogus (and part of a pattern of HHVM's G++ builds flaking but the clang builds being fine), but the Ubuntu CI job isn't being ran - probably because of the above

Reply:
Good call on the trigger, I have updated the workflow file as you suggested. It passed on the new run: https://github.com/facebook/hhvm/actions/runs/3838187863/jobs/6534350108